packages/autorest.go/test/autorest/complexmodelgroup/fake/zz_complexmodel_server.go

```diff
@@ -122,7 +122,7 @@ func (c *ComplexModelServerTransport) dispatchList(req *http.Request) (*http.Res
 	const regexStr = `/subscriptions/(?P<subscriptionId>[!#&$-;=?-\[\]_a-zA-Z0-9~%@]+)/resourcegroups/(?P<resourceGroupName>[!#&$-;=?-\[\]_a-zA-Z0-9~%@]+)/Microsoft.Cache/Redis`
 	regex := regexp.MustCompile(regexStr)
 	matches := regex.FindStringSubmatch(req.URL.EscapedPath())
-	if matches == nil || len(matches) < 2 {
+	if matches == nil || len(matches) < 1 {
```

Comment:
In this case, `subscriptionId` is a literal (`resourceGroupName` isn't).  That's why the count was decreased.